### PR TITLE
Upgrade to include Emoji 13.1 set

### PR DESCRIPTION
**Pull request by yagudaev**
Upgraded `emojilib` and `emoji-datasource` to include the latest available resources from those libraries [Emoji 13.0](https://emojipedia.org/emoji-13.0/) and [Emoji 13.1](https://emojipedia.org/emoji-13.1/).

Had a few hiccups along the way. Like this one: https://github.com/iamcal/emoji-data/issues/193.

Update to 13.1 was thanks to: https://github.com/iamcal/emoji-data/issues/192.

Finally, I'll annotate a few things in the source to call attention to during code review.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,19 +40,21 @@
...
     "babel-plugin-transform-define": "^2.0.0",
-    "emoji-datasource": "5.0.1",
-    "emojilib": "^2.2.1",
+    "emoji-datasource": "6.0.1",
+    "emojilib": "^3.0.2",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.11.2",
     "inflection": "1.10.0",
     "jest": "^24.9.0",
+    "lodash": "^4.17.21",
```
Comment (yagudaev) [inline package.json:49]: brought it for keyBy -- I'll explain below.

File: scripts/build.js [hunk 2]
```diff
@@ -135,3 +139,12 @@ module.exports = (options) => {
...
   })
 }
+
+function selectEmojisBySlug(keywordSet, data) {
+  // conversion from v2 to v3 as described in https://github.com/muan/emojilib#migrating-from-2x
+  for (const emoji in data) {
+    data[emoji]['keywords'] = keywordSet[emoji]
+  }
+
+  return keyBy(data, 'slug')
```
Comment (yagudaev) [inline scripts/build.js:149]: KeyBy needed to invert an array into a hash with a key. Looking at it again, I can probably just code it using a simple loop. Maybe even use the loop above.

**Comment by yagudaev** [comment]
Updated to support Emoji set 13.1

**Comment by computmaxer** [comment]
@EtienneLem @nolanlawson it would be great to get some updated emoji sets added!

**Comment by EtienneLem** [comment]
A new major upgrade is in the works, it will be available very soon.

**Comment by yagudaev** [comment]
Amazing to hear @EtienneLem. Love it, I've used it for [Emojination](https://emojination.io) and now [Conceptual](https://conceptual.so).

I did fork it to support this and made a few other changes for my use-case.

Happy to try the new early version and give feedback and support you any other way 😁.

Keyboard accessibility and performance when doing a search would be top of my list of wanted features.

Thanks again for taking the time to create and publish this library, it's been a massive help.

**Comment by KishanBagaria** [comment]
@yagudaev did you include 14.0 changes anywhere?

**Comment by EtienneLem** [comment]
Thanks a lot for your contribution. [v5.0.0](https://github.com/missive/emoji-mart/discussions/573) has been released and this doesn’t apply anymore.

The new version is using Emoji 14.

**Pull request closed without merging**